modules/virt-hot-unplugging-a-virtual-disk.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * virtual_machines/virtual_disks/virt-hot-plugging-virtual-disks.adoc
+
+[id="virt-hot-unplugging-a-virtual-disk_{context}"]
+= Hot-unplugging a virtual disk
+
+Hot-unplug virtual disks that you no longer want to add to or remove from a virtual machine instance while a virtual machine is running.
+
+.Prerequisites
+* You must have a running virtual machine to hot-unplug a virtual disk.
+* You must have at least one data volume or persistent volume claims (PVC) available that have been hot-plugged, using the `addvolume` command with `virtctl`.
+
+To hot-unplug a virtual disk using the CLI, use the `removevolume` command with `virtctl`.
+
+.Procedure
+
+. Run the following command:
+
+----
+$ virtctl removevolume <virtual-machine|virtual-machine-instance> --volume-name=<datavolume-or-PVC-name-to-hot-plug>
+----
+
+* The `removevolume` command deletes the attachment pod that was created to attach the hot-plugged disk. You created the attachment pod by running the `addvolume` command with `virtctl`.
```

Comment:
Again the attachment pod is an implementation detail. The removevolume doesn't actually delete the pod. It instructs the system to remove the disk. As part of that process the attachment pod is deleted (and the deletion of the pod triggers kubernetes to clean up any volumes attached to the node the VM is running on)

Reply:
Noted and removed as with the `addvolume`